docs/cvs_guidance.md

```diff
@@ -0,0 +1,44 @@
+# Controlled vocabularies guidance
+
+This doc is a placeholder for capturing notes about the rules
+which apply to our controlled vocabularies (CVs).
+It will be superseded by code which actually implements these rules
+(and code is much more precise than words, hence these docs will
+be replaced by docs documenting the code instead).
+Lots of thinking about this has been done already, and will be used to build
+out the code which implements the rules, e.g.:
+
+- https://docs.google.com/document/d/1h0r8RZr_f3-8egBMMh7aqLwy3snpD6_MrDz1q8n5XUk/edit
+- https://docs.google.com/document/d/1oLK4mWW6TX2YPrhGoLdMLcrK7vX1flU3BjiheTb2Hwk/edit
+
+## Source ID
+
+The source ID is the key way of registering a contribution to ESGF.
+The source ID then also defines other common metadata.
+Any field can be placed in source ID.
+This provides flexibility for data providers to add extra information specific to their use case.
+Some fields you may consider including are references, comment
+Having said this, the following fields are compulsory in each source ID submission:
```

Comment:
@durack1 the idea here is that many of the other fields you need in a file (realm, frequency, grid etc. vary from file to file so they can't be specified in the source ID and rather have to be added elsewhere)

Reply:
Yes that was what I was assuming. Use as much information that is already defined so we avoid defining things more than once and confusing everyone if the definitions don't match